packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/calendar.tests.js

```diff
@@ -1383,6 +1383,35 @@ QUnit.module('Preserve time component on value change', {
 
         assert.deepEqual(calendar.option('value'), new Date(2015, 8, 2, 12, 57));
     });
+
+    QUnit.test('should not switch to the earliest date after setting same value programmatically (T1279950)', function(assert) {
```

Comment:
I suggest you to make this test's name a bot more general but strongly related to the case
```suggestion
    QUnit.test('should not navigate view after new date UI select in React controlled mode (T1279950)', function(assert) {
```

Reply:
Fixed in https://github.com/DevExpress/DevExtreme/pull/29358/commits/6e2f06364df3b09fa19b6198ba960924d2ea7c69